sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/TestJsonData.scala

```diff
@@ -32,7 +32,7 @@ private[json] trait TestJsonData {
           "double":1.7976931348623157E308,
           "boolean":true,
           "null":null
-      }"""  :: Nil)
+      }"""  :: Nil, 1)
```

Comment:
Even the FileSourceStraetgy can handle empty files, it's still good to not write out empty files, that's the reason I updated these tests.